client-js/app/elements/labrad-plot.ts

```diff
@@ -135,6 +149,11 @@ export class Plot extends polymer.Base {
       area: HTMLElement, totWidth: number, totHeight: number): void {
     const p = this;
 
+    // Make room for the color bar if necessary.
+    if (p.numIndeps == 2) {
+      p.margin.right += COLOR_BAR_WIDGET_SIZE;
```

Comment:
Ahh, of course that's what it does. Not sure how I missed that after the refactor. Done.